### PR TITLE
Add JANDI

### PR DESCRIPTION
**Pull request by hohobabyp**

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -1109,3 +1109,12 @@
...
   icon: "istrolid.png"
+-
+  name: "JANDI"
+  description: "Team Communication for Businesses"
+  website: "https://www.jandi.com"
+  keywords:
+    - "messanger"
+    - "collaboration"
+  icon: "jandi.png"
+  featuredOnHomepage: true
```
Comment (paulcbetts) [inline _data/apps.yml]: Was this intentional? This means we've got an extra one-item row /cc @jlord

Comment (hohobabyp) [inline _data/apps.yml]: @paulcbetts Could you explain which node should i change?

Comment (kevinsawicki) [inline _data/apps.yml]: @hohobabyp we aren't currently adding new features apps at this time to the homepage, I've removed this line in https://github.com/electron/electron.atom.io/commit/6ce7717e21ea275f2d04f18a63eeb29f7266dc1e

It is still displayed on http://electron.atom.io/apps/

Comment (hohobabyp) [inline _data/apps.yml]: @paulcbetts Oh. I got it. Thank you.

**Pull request merged**